.github/workflows/get_release_message.yml

```
@@ -2,7 +2,7 @@ name: Get Release Message
 on: workflow_dispatch
 jobs:
   build:
-    name: Get Fonts From Icomoon
+    name: Get release message
```

IMHO, a normal user wouldn't kind of understand this. Just because we already know what it means, it's easy for us to understand. 🙂
```suggestion
    name: Get icons from Icomoon
```

---

Well, this workflow is for creating the release message though. Renaming it to "Get icons from Icomoon" would be misleading.

How about "Get features since last release"?

---

Sounds good to me. 👍